### ISSUE TITLE
Feature request: a tag property

### ISSUE THREAD

**Issue by apc518** [molleindustria/p5.play#207]
In one game I'm working on, I've added a `tag` property to the sprite class, which just an `Object` by default. This is useful any time you want to store some data about a sprite within the sprite that hasn't been defined already.

One piece of data I store in my tag is health, for instance.

In my case I created a class called `MetaObj` that had a bunch of relevant fields, then assigned a `MetaObj` to the `tag` property of my sprites. But you can do just about anything with it.

In short, it's a nice way to enable the coupling of arbitrary data with a sprite.

**Comment by quinton-ashley** [comment]
What you've discovered is actually a great feature of JavaScript! Unlike many other programming languages, in JS you can easily add properties to any object. You could even store the player's health property right on the sprite object itself.
